src/components/menu/MenuList.tsx

```diff
@@ -125,22 +139,27 @@ const MenuListComponent = () => {
     [menuProps]
   );
 
+  if (!isVisible)
+    return null
```

Comment:
Did not test it, but I guess that this will break the animation when closing the menu.
We might need to wait for the animation somehow and then remove it

Reply:
It didn't brake, you can check it